bot/exts/evergreen/trivia_quiz.py

```diff
@@ -136,9 +357,11 @@ def check(m: discord.Message) -> bool:
                 # if number of hints sent or time alerts sent is less than 2, then send one.
```

Comment:
The comment is not necessary here. It explains exactly what the code does (and doesn't mention that the hint counter increases!), which is already very clear. So I'd say just remove it

Reply:
alright!